src/component/UserProfile.js

```diff
@@ -0,0 +1,85 @@
+import React, { useEffect, useState } from "react";
+import {Text,StyleSheet,ScrollView,Image,SafeAreaView} from 'react-native';
+import { NavigationContainer } from '@react-navigation/native';
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
+import ShowUserList from "./HomePage";
+
+export default function ShowUserProfile (props){
+  return(
+    <SafeAreaView style={styles.container}>
+      <ScrollView style={styles.scrollView}>
+        <Image
+          style={styles.image}
+          source={{uri: props.route.params.item.user.avatar_url}}
+        />  
+        <Text style={{textAlign: 'center',fontSize: 20,fontFamily: 'monospace',color: 'black'}} > {props.route.params.item.user.login}</Text>
+        
+        <Text style={styles.text} > <Text style={styles.bold}>ID:</Text> {props.route.params.item.user.id}</Text>
```

Comment:
when you see deprecated sequential of variable, combine them it in one line

Reply:
I do not understand?